src/palace/manager/integration/patron_auth/saml/provider.py

```diff
@@ -369,17 +369,7 @@ def remote_patron_lookup(
         :param patron_or_patrondata: PatronData or Patron object
         :return: PatronData object if the input contains sufficient information, None otherwise
         """
-        if isinstance(patron_or_patrondata, PatronData):
-            # Return the PatronData as-is since we can't do a fresh SAML lookup
-            return patron_or_patrondata
-
-        # Convert the Patron to PatronData for admin operations
-        return PatronData(
-            permanent_id=patron_or_patrondata.external_identifier,
-            authorization_identifier=patron_or_patrondata.authorization_identifier,
-            username=patron_or_patrondata.username,
-            complete=True,
-        )
+        raise PatronLookupNotSupported()
```

Comment:
Just a note: This seems like a good starting point. We might eventually try a "cache" lookup, since we sometimes have a SAML token credential with some attributes. This could be tricky, though, since those attributes don't always align with the identifier(s) the user knows.